.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+[comment]: # (Run your command with additional informational output [-v] and try to pinpoint the problem first.)
+
+## Describe the bug
+A clear and concise description of what the bug is.
+If needed add a step by step guide to reproduce the bug.
+
+## Expected behavior
+A clear and concise description of what you expected to happen.
+
+## Command
+~~~
+Replace this line with the command you've run [e.g. truckersmp-cli -eusp]
+~~~
+
+## Terminal output
+<details><summary>Collapsed log</summary>
+
+~~~
+Replace this line with the terminal output you're getting
+when running the command with debug output [-vv]
+~~~
+</details>
+
+## Screenshots
+<details><summary>Collapsed screenshots</summary>
+
+[comment]: # (If applicable, add screenshots to help explain your problem:)
+
+![Screenshot 1](url)
```

Comment:
Reporters would need to remove this line.

I think we don't need the line because "image link" code will be inserted automatically when uploading screenshots.